src/server.ts

```diff
@@ -37,8 +42,14 @@ import { getPassportStrategy } from "./infrastructure/oidc";
       extended: true
     })
   );
+
   app.set("view engine", "html");
 
+  // Middleware to serve static assets
+  app.use("/styles", express.static(path.join(__dirname, "styles")));
+  app.use("/assets", express.static(path.resolve(__dirname, "../node_modules/govuk-frontend/assets")));
+  app.use("/node_modules/govuk-frontend", express.static(path.resolve(__dirname, "../node_modules/govuk-frontend")));
```

Comment:
This seems to pull in a lot and I'd prefer if we pulled in public assets more selectively